services/blockchain-app-registry/shared/constants.js

```diff
@@ -19,7 +19,12 @@ const KV_STORE_KEY = {
 	COMMIT_HASH_UNTIL_LAST_SYNC: 'commitHashUntilLastSync',
 };
 
+const ALLOWED_FILES = ['app.json', 'nativetokens.json'];
+const ALLOWED_FILE_EXTENSIONS = ['.png', '.svg'];
```

Comment:
1. Maybe we can reuse the allowed files defined in config?
https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/services/blockchain-app-registry/config.js#L81-L84

2. Also, maybe we can move the both list in one file (maybe config?) and then only export one variable as the concatenation of these two? Ex: `ALLOWED_META_FILES_AND_EXTENSION` which is a concatenation of `ALLOWED_FILES` and `ALLOWED_FILE_EXTENSIONS`

Reply:
> Maybe we can reuse the allowed files defined in config?
Agreed!

> Also, maybe we can move the both list in one file (maybe config?) and then only export one variable as the concatenation of these two? Ex: ALLOWED_META_FILES_AND_EXTENSION which is a concatenation of ALLOWED_FILES and ALLOWED_FILE_EXTENSIONS

Please move the constants to the config file as advised but keep ALLOWED_FILES and ALLOWED_FILE_EXTENSIONS separate since they contain different types of data.

ALLOWED_FILES can be defined as `Object.values(config.FILENAME)`